peps/pep-0725.rst

```diff
@@ -203,29 +212,73 @@ and the `Sonatype OSS Index <https://ossindex.sonatype.org/doc/coordinates>`__;
 not having to wait years before support in such tooling arrives is valuable.
 
 For concrete packages without a canonical package manager to refer to, either
-``pkg:generic/pkg-name`` can be used, or a direct reference to the VCS system
+``dep:generic/dep-name`` can be used, or a direct reference to the VCS system
 that the package is maintained in (e.g.,
-``pkg:github/user-or-org-name/pkg-name``). Which of these is more appropriate
-is situation-dependent. This PEP recommends using ``pkg:generic`` when the
-package name is unambiguous and well-known (e.g., ``pkg:generic/git`` or
-``pkg:generic/openblas``), and using the VCS as the PURL type otherwise.
+``dep:github/user-or-org-name/dep-name``). Which of these is more appropriate
+is situation-dependent. This PEP recommends using ``dep:generic`` when the
+package name is unambiguous and well-known (e.g., ``dep:generic/git`` or
+``dep:generic/openblas``), and using the VCS as the type otherwise.
 
 Virtual package specification
-'''''''''''''''''''''''''''''
+''''''''''''''''''''''''''''''
+
+PURL does not offer support for virtual or virtual dependency specification yet.
+A `proposal to add a virtual type <https://github.com/package-url/purl-spec/pull/450>`__
+is being discussed for revision 1.1.
+
+In the meantime, we propose adding a new *type* to our ``dep:`` derivative, the ``virtual``
+type, which can take two *namespaces*:
+
+- ``interface``: for components such as BLAS or MPI.
+- ``compiler``: for compiled languages like C or Rust.
 
-There is no ready-made support for virtual packages in PURL or another
-standard. There are a relatively limited number of such dependencies though,
-and adopting a scheme similar to PURL but with the ``virtual:`` rather than
-``pkg:`` scheme seems like it will be understandable and map well to Linux
-distros with virtual packages and to the likes of Conda and Spack.
+The *name* should be the most common name for the interface or language, lowercased.
+Some examples include::
 
-The two known virtual package types are ``compiler`` and ``interface``.
+  dep:virtual/compiler/c
+  dep:virtual/compiler/c++
+  dep:virtual/compiler/rust
+  dep:virtual/interface/blas
+  dep:virtual/interface/lapack
+
+Since there are a relatively limited number of such dependencies,
+it seems like it will be understandable and map well to Linux
+distros with virtual packages and to the likes of conda and Spack.
+
+.. JRG: A long shot for now. Assuming virtual PURLs would be formatted
+.. like ``pkg:virtual/{compiler,interface}/...``.
```

Comment:
We have to clean up comments before submitting the PEP revision upstream. I assume the PURL conversation won't move before then.